usersync/cookie_test.go

```diff
@@ -333,31 +385,387 @@ func ensureConsistency(t *testing.T, cookie *PBSCookie) {
 	}
 }
 
-func newTempId(uid string) uidWithExpiry {
+func newTempId(uid string, offset int) uidWithExpiry {
 	return uidWithExpiry{
 		UID:     uid,
-		Expires: time.Now().Add(10 * time.Minute),
+		Expires: time.Now().Add(time.Duration(offset) * time.Minute),
 	}
 }
 
 func newSampleCookie() *PBSCookie {
 	return &PBSCookie{
 		uids: map[string]uidWithExpiry{
-			"adnxs":   newTempId("123"),
-			"rubicon": newTempId("456"),
+			"adnxs":   newTempId("123", 10),
+			"rubicon": newTempId("456", 10),
 		},
 		optOut:   false,
 		birthday: timestamp(),
 	}
 }
 
-func writeThenRead(cookie *PBSCookie) *PBSCookie {
+func newTestCookie() (*PBSCookie, int) {
+	var mediumSizeCookie *PBSCookie = &PBSCookie{
+		uids: map[string]uidWithExpiry{
+			"key1": newTempId("12345678901234567890123456789012345678901234567890", 7),
+			"key2": newTempId("abcdefghijklmnopqrstuvwxyz", 6),
+			"key3": newTempId("ABCDEFGHIJKLMNOPQRSTUVWXYZ", 6),
+			"key4": newTempId("12345678901234567890123456789612345678901234567890", 5),
+			"key5": newTempId("aAbBcCdDeEfFgGhHiIjJkKlLmMnNoOpPqQrRsStTuUvVwWxXyYzZ", 4),
+			"key6": newTempId("12345678901234567890123456789012345678901234567890", 3),
+			"key7": newTempId("abcdefghijklmnopqrstuvwxyz", 2),
+		},
+		optOut:   false,
+		birthday: timestamp(),
+	}
+	return mediumSizeCookie, len(mediumSizeCookie.uids)
+}
+
+func newBigCookie() (*PBSCookie, int) {
+	var bigCookie *PBSCookie = &PBSCookie{
```

Comment:
How about creating a test folder and adding these different cookies there in `.txt` files and then using those in the tests rather than populating this test file with a giant map?

Reply:
Spoke with @guscarreon about this and we probably don't need this particular test case to start with and so he's gonna evaluate and remove this if not needed.